__tests__/utilityFunctions.test.js

```diff
@@ -0,0 +1,81 @@
+const fs = require('fs');
+const fileData = JSON.parse(fs.readFileSync('./user.json'));
+const { validationUserPost, addUserPost, putUserbyId, deleteUserById, getUserById, readFromUserFile } = require("../utilityFunctions");
+
+/*
+the function below i made because i thought it is maybe better to don't use hardcoded data, if it is better like this then the user and objects can be replaced by  
+randomWholeUserFromFile and randomUserNameFromFile, if not i will remove this in the next PR.
+function getRandomInt(max) {
+  return Math.floor(Math.random() * max);
+}
+const userNumber = getRandomInt(fileData.length);
+const randomWholeUserFromFile = fileData[userNumber];
+const randomUserNameFromFile = randomWholeUserFromFile.name;
+*/
+
+//this test is not perfect because i still have this issue with the append part mentioned in the utilityFunctions file.
+test('testPostAddUser', () => {
+  expect(addUserPost({
+    "name": "unitTestUser",
+    "height": 190,
+    "hair": "Black"
+  })).toBe(true);
+});
+
+// this test is not perfect because i still have this issue with the append part mentioned in the utilityFunctions file.
+test('testPutUserById', () => {
+  expect(putUserbyId({
```

Comment:
All cases could improve by storing the variable first.

Reply:
yeah that is indeed more readable, done!